dev/fattest.simplicity/src/componenttest/custom/junit/runner/FATRunner.java

```diff
@@ -622,46 +622,53 @@ private String getFFDCHeader(RemoteFile ffdcLogFile) throws Exception {
     public List<String> getExpectedFFDCAnnotationFromTest(FrameworkMethod m) {
 
         ArrayList<String> annotationListPerClass = new ArrayList<String>();
-        ExpectedFFDC ffdc = m.getAnnotation(ExpectedFFDC.class);
-        if (ffdc != null) {
-            String[] exceptionClasses = ffdc.value();
-            for (String exceptionClass : exceptionClasses) {
-                annotationListPerClass.add(exceptionClass);
+
+        Set<ExpectedFFDC> ffdcs = new HashSet<ExpectedFFDC>();
```

Comment:
I don't think a set is necessary here since there will only ever be one `ExpectedFFDC`